pydoctor/model.py

```diff
@@ -77,7 +77,27 @@ class PrivacyClass(Enum):
     HIDDEN = 0
     PRIVATE = 1
     VISIBLE = 2
+    
+class KindClass(Enum):
+    """
+    L{Enum} containing values indicating the possible object types.
 
+    @note: Presentation order is derived from the enum values
```

Comment:
I still think it would be simpler to use `auto()` and remove the `-` from the sorting.

Reply:
I think auto() makes it more complicated to understand . With numbers it’s clear which will pop up first. Also it keeps it consistent with other enigma

Reply:
The other enums were added when using the `enum34` compatiblity package when we were still supporting Python 2.7. If I added them today, I'd probably use `auto()`.